examples/webvr_rollercoaster.html

```diff
@@ -165,7 +165,8 @@
 
 			var geometry = new THREE.CylinderBufferGeometry( 10, 10, 5, 15 );
 			var material = new THREE.MeshLambertMaterial( {
-				color: 0xff8080, shading: THREE.FlatShading
+				color: 0xff8080,
+				//flatShading: true // Lambert does not support flat shading
```

Comment:
@mrdoob Note this. Either use `MeshPhongMaterial`, or use `Geometry` and call `geometry.computeFlatVertexNormals();`.